### PR TITLE
Write-up for Pythonic API quickstart

### PR DESCRIPTION
**Pull request by alex-ozdemir**

### CODE REVIEW

File: examples/api/python/pythonic/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Notes
+
+These files are copied from [the pythonic API's](TODO).
```
Comment (mpreiner) [inline examples/api/python/pythonic/README.md:3]: Should this be the URL of the pythonic API repo?

File: docs/conf.py.in [hunk 2]
```diff
@@ -160,6 +160,11 @@ examples_types = {
                 'lang': 'java',
                 'group': 'java'
         },
+        '^<examples>.*pythonic.*\.py$': {
+                'title': 'Python (pythonic)',
+                'lang': 'python',
+                'group': 'py-regular'
```
Comment (mpreiner) [inline docs/conf.py.in:166]: Why is this `py-regular` and before it was `py-pythonicapi`?

File: examples/api/python/pythonic/README.md (hunk 1 above)
Comment (alex-ozdemir) [reply examples/api/python/pythonic/README.md]: Yes, good catch.

File: docs/conf.py.in (hunk 2 above)
Comment (alex-ozdemir) [reply docs/conf.py.in]: That is a mistake. Good catch!

**Review by mpreiner — APPROVED** [review]

**Pull request merged**